lib/init.lua

```diff
@@ -46,4 +47,9 @@ apply(Roact, {
 	getGlobalConfigValue = GlobalConfig.getValue,
 })
 
+apply(Roact, {
+	getCollectedStats = Instrumentation.getCollectedStats,
+	clearCollectedStats = Instrumentation.clearCollectedStats,
+})
```

Comment:
That's fair. I've addressed it

Reply:
Would it be okay to expose them as a table, something like `Roact.Instrumentation.getCollectedStats`?

Reply:
Heh, sounds my original implementation.  @LPGhatguy It would be good to expose them in some way so that there's some utility to these changes, especially for the sake of the other tooling I'm working on.  Let me know what you think.

Reply:
Putting them at `Roact.Instrumentation.getCollectedStats` would be fine.

The original version exposed them as `Roact.getCollectedStats`, that was the problematic variant!

Reply:
The original version (a few commits ago) actually had the whole `Instrumentation` object exposed, but it makes more sense to expose it indirectly.